.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "kani"]
```

Comment:
Tracking issue to remove this once we publish the Kani crate

Reply:
#1639